config_examples/k8scombined.yaml

```diff
@@ -68,8 +97,6 @@ processors:
         statements:
           - set(attributes["k8s.cluster.name"], "${env:CLUSTER_NAME}")
   k8sattributes:
-    filter:
-      node_from_env_var: K8S_NODE_NAME
```

Comment:
can we then get rid of the env variable entirely?

Reply:
no, we still need it for kubeletstats receiver for node metrics